docs/manual/resource_tracking.md

```diff
@@ -155,3 +155,13 @@ We link as well the pending instance(given by `squest.instance.id`) to this reso
         status_code: 201
         body_format: json
 ```
+
+## Over commitment
+
+Over commitment is available on all resource pool attributes for consumers and producers.
+
+The over commitment allows you to specify whether resource pools produce/consume more or less than expected.
+
+The most common case is CPU/vCPU:
+
+If a host has two eight core processors and hyperthreading is enabled, that host will produces thirty-two vCPUs (8 cores x 2processors x 2 threads per core).
```

Comment:
then you can configure the over commitment on the CPU produced attribute to `2` to correspond to this behavior.